src/amo/actions/reviews.js

```diff
@@ -584,3 +586,31 @@ export const updateAddonReview = ({
     payload: { body, errorHandlerId, rating, reviewId },
   };
 };
+
+export const ABORTED: 'aborted' = 'aborted';
+export const SAVED_RATING: 'saved-rating' = 'saved-rating';
+export const SAVED_REVIEW: 'saved-review' = 'saved-review';
+export const STARTED_SAVE_RATING: 'started-save-rating' = 'started-save-rating';
```

Comment:
Why did you switch to lowercase for flow types/values?

Reply:
The lower case values are just what gets saved to Redux state. I don't think they need to be upper case. This is also consistent with how we do it elsewhere:

https://github.com/mozilla/addons-frontend/blob/c698d47a25d7f0b93626f0ecff2691817d80061e/src/amo/constants.js#L2-L9

https://github.com/mozilla/addons-frontend/blob/c698d47a25d7f0b93626f0ecff2691817d80061e/src/amo/reducers/recommendations.js#L13-L16

Reply:
well, it would be more consistent if we had only one case for all redux constants.

Reply:
I didn't really want to use constants at all here because I thought Flow would be enough to prevent mistakes. However, Flow won't show an error if you make an equality comparison with a bogus string like:

```js
if (flashMessage === 'started-typo') {
}
```

...which may be a Flow bug but I can think of why they may not want it to be an error. The constants will help prevent mistakes like that.